WalletWasabi/WabiSabi/Backend/Rounds/CoinJoinStorage/CoinJoinIdStore.cs

```diff
@@ -0,0 +1,43 @@
+using NBitcoin;
+using System.IO;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.WabiSabi.Backend.Rounds.CoinJoinStorage;
+
+public class CoinJoinIdStore : ICoinJoinIdStore
+{
+	public InMemoryCoinJoinIdStore InMemoryCoinJoinIdStore { get; set; }
+	public string CoinJoinsFilePath { get; set; }
+	private object FileWriteLock { get; set; } = new();
+
+	public CoinJoinIdStore() : this(string.Empty)
+	{
+	}
+
+	public CoinJoinIdStore(string ww2CoinjoinsFilePath)
```

Comment:
Use naming of coinJoinIdStoreFilePath because of consistency with coordinatorParameters